src/components/AddItemForm.js

```diff
@@ -69,9 +70,12 @@ class AddItemForm extends React.Component {
         </label>
         <br />
         <input className="submit-btn" type="submit" value="Submit" />
+        <FormAlert userSubmit={this.state.name} />
```

Comment:
You may not even need a separate component for this unless we were going to use a single component for any errors/messages across the app.